.changeset/serious-melons-sip.md

```diff
@@ -0,0 +1,6 @@
+---
+'@emotion/native': patch
+'@emotion/styled': patch
+---
+
+Fixed TypeScript false positive with `styled` components
```

Comment:
I don't think this changeset is particularly useful - in general, we try to express user-facing changes, and AFAIK what has been published from `next` to npm did not suffer from many problems and this doesn't quite even explain what it has fixed/changed/added. 

If this is related to the introduced changes in this PR then this can be safely removed because the core of the change is already explained in the other changeset. But maybe I've misunderstood intention here.

Reply:
This changeset was specifically for the `InferredAdditionalProps` change I added. Since it shouldn't have a user-visible facing effect I'll remove it.

Reply:
I have concerns regarding this additional type param - would prefer solving this without introducing it. I plan to release v11 today but as this is type-only change I think it will be OK to just release it in a minor version later when we settle on a final solution.